common/src/webida/plugins/editors/workbench-panels.js

```diff
@@ -112,8 +112,13 @@ define([
                 dialog.destroyRecursive();
             }
         });
-        dialog.setContentArea('<span> File "' + file.name  + '" has unsaved changes. </span>' +
-                '<span> Save and '+ title + ' this file? </span>');
+
+        var name = file.name;
+        if (title === 'Quit') {
```

Comment:
I think it's better 'Quit' is defined as constant and re-used in creating a dialog and checking.